Increase button contrast on main webiste and app website

# Feature request

## Is your feature request related to a problem? Please describe.

The text on the "accent" buttons on these two websites don't have sufficient contrast with their background, and do not meet WCAG standards for accessible text.

![Supabase website's landing page, showing firefox' developer tools analysis on the accent buttons, which have a contrast of 2.62](https://user-images.githubusercontent.com/70592708/194485596-ab655029-82f6-417b-b98c-591a3f9d43cd.png)

## Describe the solution you'd like

Changing the white text to a much darker/black text would increase readability and comply with WCAG guidelines.

![Supabase website's landing page, showing firefox' developer tools analysis on the accent buttons with text changed to black, which have a contrast of 6.33 ](https://user-images.githubusercontent.com/70592708/194486782-190662db-50ec-45b1-8f4f-b9b3b68513cc.png)
![image](https://user-images.githubusercontent.com/70592708/194487032-f67fe5bd-5161-45fe-8df8-ee1faba77b4a.png)

## Describe alternatives you've considered

Using background color works too, and keeps consistency with overall theme.

## Additional context

This problem was already addressed in #329 and apparently fixed by #435, but the changes must've been reversed at some point.

I can't seem to be able to run the website's developer server. I cd into apps/www, run `npm i` and `npm run dev` I get the following error:

```bash
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /home/rafavls/Contributing/supabase/apps/www/.env
info  - automatically enabled Fast Refresh for 1 custom loader
error - ../../node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!../../node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./styles/index.css
Error: Cannot find module 'config/postcss.config'
Require stack:
- /home/rafavls/Contributing/supabase/apps/www/postcss.config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/lib/find-config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/blocks/css/index.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/index.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack-config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/dev/hot-reloader.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/dev/next-dev-server.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/next.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/lib/start-server.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/cli/next-dev.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/bin/next
```

Any clue on how to solve this?

--- a/packages/ui/src/lib/theme/defaultTheme.ts
+++ b/packages/ui/src/lib/theme/defaultTheme.ts
@@ -441,9 +441,9 @@ export default {
     container: 'inline-flex font-medium',
     type: {
       primary: `
-        bg-brand-fixed-800 hover:bg-brand-fixed-900
+        bg-brand-fixed-1100 hover:bg-brand-fixed-1000
         text-white
-        bordershadow-brand-fixed-800 hover:bordershadow-brand-fixed-1000
+        bordershadow-brand-fixed-1000 hover:bordershadow-brand-fixed-900 dark:bordershadow-brand-fixed-1000 dark:hover:bordershadow-brand-fixed-1000
         focus-visible:outline-brand-600
       `,
       secondary: `

